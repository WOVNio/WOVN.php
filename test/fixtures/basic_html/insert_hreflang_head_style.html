<html>
<head class="display:none"><script type="text/javascript" src="fake.js"></script><link rel="stylesheet" src="fake.css"></head>
<body>
<p>Lorem ipsum</p>
</body>
</html>
