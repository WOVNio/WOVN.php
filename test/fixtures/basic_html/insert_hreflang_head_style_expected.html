<html>
<head class="display:none"><link rel="alternate" hreflang="en" href="http://my-site.com/en/"><link rel="alternate" hreflang="vi" href="http://my-site.com/vi/"><script src="//j.wovn.io/1" data-wovnio="key=123456&amp;backend=true&amp;currentLang=ja&amp;defaultLang=ja&amp;urlPattern=path&amp;langCodeAliases=[]&amp;langParamName=wovn" data-wovnio-info="version=WOVN.php_VERSION" async></script><script type="text/javascript" src="fake.js"></script><link rel="stylesheet" src="fake.css"></head>
<body>
<p>Lorem ipsum</p>
</body>
</html>
